core/src/main/java/org/fao/geonet/kernel/metadata/DefaultStatusActions.java

```diff
@@ -352,8 +342,8 @@ protected void sendEmail(String sendTo, String subject, String status, String ch
             context.info("Would send email \nTo: " + sendTo + "\nSubject: " + subject + "\n Message:\n" + message);
         } else {
             MailSender sender = new MailSender(context);
-            sender.sendWithReplyTo(host, Integer.parseInt(port), username, password, useSSL, useTLS, from, fromDescr,
-                sendTo, null, replyTo, replyToDescr, subject, message);
+            sender.sendWithReplyTo(host, Integer.parseInt(port), username, password, useSSL, useTLS,
```

Comment:
We should think about some refactoring of all this information (not required in this PR). Would be good in some moment to take this information to a custom class MailSenderInfo or similar.